distributed-calculator/README.md

```diff
@@ -616,13 +616,18 @@ The code below shows calls to the "add" and "subtract" services via the Dapr URL
 const daprUrl = `http://localhost:${daprPort}/v1.0/invoke`;
 
 app.post('/calculate/add', async (req, res) => {
-  const addUrl = `${daprUrl}/addapp/method/add`;
-  req.pipe(request(addUrl)).pipe(res);
+  const appUrl = `${daprUrl}/addapp/method/add`;
```

Comment:
Any reason we are keeping this separately?

Reply:
We can inline